### PR TITLE
Update playlist-editor.md

### PR DESCRIPTION
**Pull request by leoj3n**
Fix typos, improve sentences.

### CODE REVIEW

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md [hunk 1]
```diff
@@ -488,18 +488,18 @@ In this section, we will:
   $(element).on('dropon', function(ev, drop, drag) {...});
   ```
 
-  Notice that `drag` is the 3nd argument to the event.  You can listen to
-  `drop` events in [can-stache] and pass the `drag` argument to a function like:
+  Notice that `drag` is the 2nd argument to the event.  You can listen to
```
Comment (chasenlehara) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: I think the arguments are `ev, drop, drag` so `drag` is the third argument. Am I reading the code incorrectly?

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md [hunk 2]
```diff
@@ -488,18 +488,18 @@ In this section, we will:
...
+  `drop` events in [can-stache], and pass the `drag` argument to a function, like:
 
   ```html
   ($dropon)="addVideo(%arguments[2])"
   ```
 
 - You will need to associate the drag objects with the video being
-  dragged so when a `drop` happens you can know the video being dropped. The
+  dragged so, when a `drop` happens, you can know which video is being dropped. The
```
Comment (chasenlehara) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: This is correct, and we could probably make it better by saying “…dragged so you can know which video is being dropped when a `drop` happens.”

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md [hunk 3]
```diff
@@ -685,7 +685,7 @@ In this section, we will:
 
 - Use [https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt] to prompt a user for a simple string value.
 
-- YouTube only allows you to create a playlist and then add items to it.
+- YouTube _only_ allows you to create a playlist, and then add items to it.
```
Comment (chasenlehara) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: The comma seems unnecessary.

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md [hunk 4]
```diff
@@ -738,9 +738,9 @@ In this section, we will:
   ```
 
   When a callback to `.then` returns a promise, `.then` returns a promise that resolves
-  after the inner promise is resolved.
+  after the _inner_ promise has resolved.
```
Comment (chasenlehara) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: _is_ should be used or _has been_

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md [hunk 5]
```diff
@@ -765,6 +765,6 @@ Update the `JS` tab to:
 @sourceref ./8-create-playlist.js
 @highlight 12-19,99-141,only
 
-Congrats! You've completed this guide.
+Congrats! If you've completed this recipe, you now have your very own YouTube Playlist Editor.
```
Comment (chasenlehara) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: I think the optimism of the original text is more encouraging. Maybe _Congrats! You now have your very own YouTube Playlist Editor._

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md (hunk 1 above)
Comment (leoj3n) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: It should actually be pointing out the `drop` argument, I think. This is based on the fact that the `drag` event was already covered before this point. On line 386 of this markdown file, there is a very similar sentence:

>  Notice that `drag` is the 2nd argument to the event.  You can listen to
>  `drag` events in [can-stache] and pass the `drag` argument to a function like:

Unfortunately I can't link a line of a markdown file on GitHub.

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md (hunk 2 above)
Comment (leoj3n) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: ✅ Change made.

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md (hunk 3 above)
Comment (leoj3n) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: ✅ Change made.

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md (hunk 4 above)
Comment (leoj3n) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: ✅ Change made to "has been".

File: docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md (hunk 5 above)
Comment (leoj3n) [inline docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md]: ✅ Change made.

**Pull request merged**